models/enterprise_sonic/interfaces/overridden_example_01.txt

```diff
@@ -5,25 +5,36 @@
 #
  show interface status | no-more
 ------------------------------------------------------------------------------------------
-Name                Description         Admin          Oper           Speed          MTU
+Name                Description         Admin     Oper      AutoNeg     Speed        MTU
 ------------------------------------------------------------------------------------------
-Ethernet0           E0                  up                            100000         9100
-Ethernet4           E4                  up                            100000         9100
-Ethernet8           E8                  down                          100000         9100
-Ethernet12          -                   down                          1000           9100
-Ethernet16          -                   down                          100000         9100
+Ethernet0           E0                  up                              100000       9100
+Ethernet4           E4                  up                              100000       9100
+Ethernet8           E8                  down                            100000       9100
+Ethernet12          -                   down                            1000         9100
+Ethernet16          -                   down                            100000       9100
+
+ show running-configuration interface Ethernet 8
+!
+interface Ethernet8
+ mtu 9100
+ speed 100000
+ shutdown
 
 - name: Configure interfaces
```

Comment:
Please modify this example to show that "overridden" state removes all configuration for the resource module, and installs only the configuration specified in the playbook. It isn't necessary to show all Ethernet interfaces, but including at least one or two examples of configuration that is removed from interfaces not specified in the playbook would be important to show. 

Note: This resource module is actually a good example of where NOT to use "overridden" state or, at least, where to use it with extreme caution.

Reply:
Based on the most recent updates, this issue can be marked "Resolved".